Makefile

```diff
@@ -27,6 +27,17 @@ fast-install:
 	cp epidose/device/shutdown_epidose.sh \
 	        epidose/device/upload_seeds_d.sh \
 		epidose/device/update_filter_d.sh \
+		epidose/device/update_software_d.sh \
 		epidose/device/util.sh \
 		epidose/device/wps_scanner_d.sh $(OPT)/bin/
 	supervisorctl restart epidose:*
+
+# Install after software update are obtained from the production branch
+remote-update:
+	cp -r epidose dp3t $(SITE)/
+	cp epidose/device/shutdown_epidose.sh \
+	        epidose/device/upload_seeds_d.sh \
+		epidose/device/update_software_d.sh \
+		epidose/device/util.sh \
+		epidose/device/wps_scanner_d.sh $(OPT)/bin/
+	supervisorctl restart epidose:beacon_rx epidose:beacon_tx epidose:upload_seeds epidose:update_filter epidose:wps_scanner
```

Comment:
I realize that you can't restart the update process.  If the update software requires updating, then the update script can issue a reboot command, no?